Content.Shared/Interaction/SharedInteractionSystem.cs

```diff
@@ -1282,9 +1283,24 @@ public bool IsAccessible(Entity<TransformComponent?> user, Entity<TransformCompo
             if (_containerSystem.IsInSameOrParentContainer(user, target, out _, out var container))
                 return true;
 
+            // Begin DeltaV - allow organs to have interactible UIs
+            if (IsOrganInUser(user, target))
+                return true;
+            // End DeltaV - allow organs to have interactible UIs
+
```

Comment:
use AccessibleOverrideEvent

Reply:
using what component on the user? there's no "has augments" component & shitmed has no bookkeeping for muliple organs providing the same component

Reply:
make one with a hashset of augment entities, when an augment is installed or removed add or remove from the hashset, removing if its empty

Reply:
that feels like it's going to be prone to data redundancy bugs

Reply:
container events get raised before deletion and body system handles container events so handling body events should Just Werk and be resilient to augments randomly being deleted